packages/cli/src/commands/stream/create.ts

```diff
@@ -0,0 +1,108 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+import { parseContent, parseControllers } from '../../utils'
+
+type Flags = CommandFlags & {
+  determinitic?: boolean
+  'only-genesis'?: boolean
+
+  did?: string
+  controllers?: string
+}
+
+export default class Create extends Command<
+  Flags,
+  {
+    schema: string
+    content: string
+  }
+> {
+  static description = 'Create a new Stream'
+
+  static args = [
+    {
+      name: 'schema',
+      description: 'the StreamId of the schema to use',
+      required: true,
+    },
+    {
+      name: 'content',
+      description: 'the Stream body',
```

Comment:
Rather than having this `parseContent` utility, I think it's simpler to provide a `parse` function in the argument definition directly, for example:

```suggestion
      description: 'the Stream body',
      parse: JSON.parse,
```

I find it makes it easier to quickly identify the expected inputs, rather than having to go through the code logic.